Automattic/jetpack
# Add/boost recommendation

This PR adds a recommendation for Jetpack Boost into the recommendations flow. This recommendation shows up after a new page is published on a site that is not already using Jetpack Boost.

#### Changes proposed in this Pull Request:

* Adds a new recommendation for Jetpack Boost in the recommendation flow

#### Other information:


#### Jetpack product discussion


pbNhbs-3io-p2

#### Does this pull request change what data or activity we track or use?


This PR does not add any new events/ data. Existing events that are already in place for other recommendations will be used to track interactions with this new recommendation.

#### Testing instructions:






* On your Jetpack test site, start by resetting the Jetpack options. If you are using a development version of Jetpack, this option should be available in the footer on the Jetpack Dashboard.
* Either publish a new page or switch an existing page to "draft" and then back to "publish"
* Navigate back to the Jetpack dashboard, you should see a red "1" bade next to the "Recommendations" navigation item indicating that a new recommendation is available.
![Screen Shot 2022-08-10 at 11 35 56 AM](https://user-images.githubusercontent.com/18016357/183954260-1c81c060-005d-4afa-9116-f91872146d96.png)

* Click on the recommendations navigation item. You should now see the recommendation for Jetpack Boost. There should be a green "New" badge showing on this step.
![Screen Shot 2022-08-16 at 12 01 28 PM](https://user-images.githubusercontent.com/18016357/184926773-c887be78-8bef-4336-8c1e-8c3a825a3ba0.png)

* Click on the CTA to install Jetpack Boost. This should take you to the summary screen where "Jetpack Boost" should show under enabled recommendations ( this may take a second or two if the plugin is still activating in the background )
![Screen Shot 2022-08-10 at 11 36 36 AM](https://user-images.githubusercontent.com/18016357/183954565-c319d242-ab0d-409f-b3f9-cb4415a58ec9.png)

* Click on the "Settings" button next to the Boost recommendation on the summary screen. This should take you to the Jetpack Boost dashboard.
* Confirm that any new text is properly translated.
* Confirm that all links on the new recommendation step work and make sense.
